lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -6533,7 +6533,13 @@ namespace Js
         TryFinally([&]()
         {
             //The array is a continuous array if there is only one segment
-            if (startSeg->next == nullptr) // Single segment fast path
+            if (startSeg->next == nullptr
+                // If this flag is specified, we want to improve the consistency of our array sorts
+                // by removing missing values from all kinds of arrays before sorting (done here by
+                // using the copy-to-one-segment path for array sorts) and by using a stronger sort
+                // comparer than the spec requires (done in CompareElements).
+                && !Js::Configuration::Global.flags.StrongArraySort
```

Comment:
can you remind me where will this flag be enabled?

Reply:
This flag is intended to be used when looking at a bug; if it's something where repro/not repro differs on -forcearraybtree, this flag can identify if it's the known issue of multiple search orders being equivalent, or if it's a new, more interesting issue.